linear_algebra/code/include/linear_algebra.hpp

```diff
@@ -39,7 +39,7 @@
     #undef LA_USE_MDSPAN
 #endif
 #ifdef LA_USE_MDSPAN
-    #include <mdspan>
+    #include <experimental/mdspan>
```

Comment:
This needed to be changed because this is how the authors of MDSpan have set up their target to be included.  They expect users to include the "experimental" prefix.